dpnp/dpnp_iface_mathematical.py

```diff
@@ -1750,14 +1725,27 @@ def subtract(
 
```

Comment:
and here could you please remove `out` from `Limitations` block also

Reply:
done